app/core/Engine/Engine.ts

```diff
@@ -619,6 +621,29 @@ export class Engine {
           target,
         ),
       getClientCryptography: () => ({ pbkdf2Sha512: pbkdf2 }),
+      getPreferences: async () => {
+        const {
+          securityAlertsEnabled,
+          useTransactionSimulations,
+          useTokenDetection,
+          privacyMode,
+          useNftDetection,
+          displayNftMedia,
+          isMultiAccountBalancesEnabled,
+        } = this.getPreferences();
+        const locale = await StorageWrapper.getItem(LANGUAGE);
```

Comment:
```suggestion
        const locale = I18n.locale;
```
Seems that this is used elsewhere in the codebase instead. IMO we should avoid having to fetch it from storage

Reply:
Ok cool, I'll update